src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/RenderInputParams/index.tsx

```diff
@@ -4,28 +4,32 @@ import { useField } from 'react-final-form'
 import Row from 'src/components/layout/Row'
 
 import InputComponent from './InputComponent'
+import { generateFormFieldKey } from '../utils'
+import { AbiItemExtended } from 'src/logic/contractInteraction/sources/ABIService'
 
-const RenderInputParams = () => {
+const RenderInputParams = (): JSX.Element => {
```

Comment:
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#useful-react-prop-type-examples

see `JSX.Element vs React.ReactNode` section

and then https://github.com/facebook/create-react-app/pull/8177

the typings are so confusing 🤦